libs/platform/core/src/services/meta/meta.service.ts

```diff
@@ -0,0 +1,44 @@
+export const MetaService = 'oryx.MetaService';
```

Comment:
I'm not really sure about the name of the service as long as we can add whatever we want here, there are no restrictions. And at the same time we can only add it to the `head`. So for now it more looks like a `HeadDOMService` or something.